questions/javascript-questions.md

```diff
@@ -1,21 +1,19 @@
 # JS Questions:
 
-* Explain event delegation
-* Explain how `this` works in JavaScript
-* Explain how prototypal inheritance works
-* What do you think of AMD vs CommonJS?
-* Explain why the following doesn't work as an IIFE: `function foo(){ }();`.
```

Comment:
This seems like more of a trivia question than something relevant to most environment at present. in 2018. you could have five years of professional web development experience without having to use an IIFE.

Reply:
I would suggest changing the 3rd question to: 
```suggestion
* What do you think of ESM vs CommonJS?
```
because AMD was pretty much replaced by ES modules (or ES6/ES2015 modules) and is rarely used nowadays.
Or alternatively:
```suggestion
* Where are ESM and CJS modules used?
```